src/dsverifier.cpp

```diff
@@ -2597,6 +2981,62 @@ void extract_data_from_ss_file()
   _controller.inputs[lines][columns] = std::stod(str_bits);
   str_bits.clear();
 
+  if(dsv_strings.desired_property == "SETTLING_TIME")
```

Comment:
Can you please use the `properties` array for "SETTLING_TIME" (following the DSVerifier standard)?

See: https://github.com/ssvlab/dsverifier/blob/master/src/dsverifier.cpp#L81